packages/squiggle-lang/src/fr/tag.ts

```diff
@@ -116,7 +117,9 @@ export const library = [
   }),
   maker.make({
     name: "description",
-    examples: [],
+    description: `Adds a description to a value. This is useful for documenting what a value represents, or how it was calculated.
+    
+*Note: Descriptions provided as tags look like docstrings, but are not docstrings. Docstrings don't get exported with the variable into other files, but Tag descriptions do.*`,
```

Comment:
Can we just deprecate docstrings? I don't think many people used them, and descriptions can replace them completely.

Reply:
Yea, I like that. Will remove this comment then.